### ISSUE TITLE
Moving out of the bottom in widget is accidentally making the user to modify the endpoint

### ISSUE THREAD

**Issue by diegoulloao** [darrenburns/posting#18]
Posting provides keybindings for moving up/down using the `j` and `k` keys correspondingly, which is very good, although I can observe an annoying behavior which is the user moving out the bottom of the widget causing accidentally change the endpoint as recorded in the screenshot:

https://github.com/darrenburns/posting/assets/45423661/e6c8dd2a-f736-453b-98f1-40c0fce02476

IMO, the bottom limit of the widget should be blocked, in that way the only way to move to the endpoint field/widget is using the `tab` or `shift+tab` key combination.

This is very subjective dough.
Just dropping the issue here in case it can be considered.

**Comment by darrenburns** [comment]
I agree! It bit me a couple of times - I'll change the behaviour here.
